pocs/observatory.py

```diff
@@ -148,12 +149,15 @@ def status(self):
                 status['observation']['field_ha'] = self.observer.target_hour_angle(
                     t, self.current_observation.field)
 
+            evening_astro_time = self.observer.twilight_evening_astronomical(t, which='next')
```

Comment:
What does next produce if at midnight? Tomorrow night?

Reply:
Correct. The default is `nearest`, which would go backwards. With `next` we always get the upcoming.